src/main/haskell/kore/test/Test/Kore/IndexedModule/MockMetadataTools.hs

```diff
@@ -39,6 +40,7 @@ makeMetadataTools symbolOrAliasSorts attr isSubsortOf =
         , sortAttributes = const functionAttributes
         , symbolOrAliasSorts = symbolOrAliasSorts
         , isSubsortOf = \first second -> (first, second) `elem` isSubsortOf
+        , subsorts = const Set.empty
```

Comment:
makeMetadataTools shoud get a table/graph/somehting on top of which it should compute `isSubsortOf` and `subsorts`, otherwise our code would get contradictory information in tests. Alternately, `isSubsortOf` could work as it currently does, but `makeMetadataTools` should also get a lookup table or something for `subsorts` and we would rely on the caller to keep them consistent.